src/question/models.py

```diff
@@ -1,24 +1,60 @@
 from django.db import models
 from taggit.managers import TaggableManager
 from taggit.models import TagBase
-
-from common.models import AbstractComment, Votes
+from django.core.exceptions import ValidationError
+from common.models import AbstractComment
```

Comment:
I think, It would be prettier with one line gap between third party modules and our modules imports.

Reply:
Fixed in https://github.com/DevTeamSCH/vikoverflow-backend/pull/11/commits/e63e5aa44011d4d34b60f42930ed1ac657d5aa48.